### PR TITLE
performance_presentation

### PR DESCRIPTION
**Pull request by DominiqueMakowski**
Drafted a short announcement for performance. What do you guys think?

### CODE REVIEW

**Comment by strengejacke** [comment]
Looks good! I would also add one of the "original" functions like R2 or ICC as well, else readers might think that the induces are only available in the comprising `model_performance()`.

**Comment by DominiqueMakowski** [comment]
right, do you have any particular examples -that you'd like to present?

**Comment by DominiqueMakowski** [comment]
Looks good to me ☺️ 

If you guys greenlight this I can build and merge tomorrow morning (I've noticed that when I merge in the morning, it appears in the afternoon but on the top-front page of r-bloggers, not sure if real pattern or coincidence though 😅)

@pdwaggoner @mattansb

**Comment by mattansb** [comment]
Looks great!

File: content/posts/performance_presentation.Rmd [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+The goal of [`performance`](https://github.com/easystats/performance) is to provide lightweight tools to **assess and check the quality of your model**. It includes functions such as [`R2`](https://easystats.github.io/performance/reference/r2.html) for many models (including logistic, mixed and Bayesian models), [`ICC`](https://easystats.github.io/performance/reference/icc.html) or helpers to check [`convergence`](https://easystats.github.io/performance/reference/check_convergence.html), [`overdipsersion`](https://easystats.github.io/performance/reference/check_overdispersion.html) or [`zero-inflation`](https://easystats.github.io/performance/reference/check_zeroinflation.html). See the **list of functions** [here](https://easystats.github.io/performance/reference/index.html).
+
+`performance` can be installed as follows:
+
+```{r, eval = FALSE, message=FALSE, warning=FALSE}
+install.packages("performance")  # Install the package
+library(performance)  # Load it
+```
+```{r message=FALSE, warning=FALSE, include=FALSE}
+library(performance)
```
Comment (pdwaggoner) [inline content/posts/performance_presentation.Rmd:42]: Not sure if this is a typo, but loading the library appears twice here.

File: content/posts/performance_presentation.Rmd [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+library(rstanarm)
+junk <- capture.output(model <- stan_glmer(Sepal.Width ~ Petal.Length + (1|Species), data = iris))
+model_performance(model)
+```
+
+
+
+More details about `performance`'s features are comming soon, stay tuned ;)
+
+- **Don't forget to check out the** [**documentation here**](https://easystats.github.io/performance/) **for more!**
```
Comment (pdwaggoner) [inline content/posts/performance_presentation.Rmd]: I would bump this up a line (i.e., over "More details about..." in line 82), as line 82 feels like a conclusion, but this line feels like a reminder that should come before the conclusion. Just a thought.

File: content/posts/performance_presentation.Rmd [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+
+## Help us
```
Comment (pdwaggoner) [inline content/posts/performance_presentation.Rmd]: I would maybe change this to something a little softer like, "Contribute", to make it seem less like we *really* need some help, and frame it more like we are developing exciting tools quickly and welcome ideas on contributing to this rapidly expanding ecosystem of tools and packages. Again, only a thought here.

File: content/posts/performance_presentation.Rmd (hunk 2 above)
Comment (DominiqueMakowski) [reply content/posts/performance_presentation.Rmd]: you're right!

File: content/posts/performance_presentation.Rmd (hunk 1 above)
Comment (DominiqueMakowski) [reply content/posts/performance_presentation.Rmd:42]: No, one is just for display (with the install.package function), the other one actually loading the thing :)

File: content/posts/performance_presentation.Rmd (hunk 3 above)
Comment (DominiqueMakowski) [reply content/posts/performance_presentation.Rmd]: I agree ☺️

File: content/posts/bayestestR_presentation.Rmd [hunk 4]
```diff
@@ -67,15 +67,15 @@ Results can also be plotted, to get a better impression of the posterior distrib
...
+- **Don't forget to check out the** [**documentation here**](https://github.com/easystats/bayestestR#documentation) **for more!**
 
-More details about `bayestestR`'s features are comming soon, stay tuned ;)
 
-- **Don't forget to check out the** [**documentation here**](https://github.com/easystats/bayestestR#documentation) **for more!**
+More details about `bayestestR`'s features are comming soon, stay tuned ;)
 
 
-## Help us
+## get involved
```
Comment (pdwaggoner) [inline content/posts/bayestestR_presentation.Rmd]: Great change here! Should it be capitalized ("Get Involved")? Not sure how you're doing the other subtitles. But I like "get involved" much better, for whatever that's worth.

Comment (DominiqueMakowski) [reply content/posts/bayestestR_presentation.Rmd]: fixed thanks :)

**Comment by DominiqueMakowski** [comment]
I've updated by mistake bayestestR presentation instead of performance. Fixed now.

**Pull request merged**